WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -434,6 +435,33 @@ public static float getUploadProgressForMedia(MediaModel mediaModel) {
         return Collections.emptyList();
     }
 
+    public static boolean isPendingOrInProgressMediaUpload(MediaModel media) {
```

Comment:
Since this is `public`, we should either mark `media` as `@NonNull`, or null-check it before we hand it off to `MediaUploadHandler` (or do this inside `MediaUploadHandler.isPendingOrInProgressMediaUpload()`).